templates/blog/index.html

```diff
@@ -0,0 +1,18 @@
+{% extends "base.html" %}
+
+{% block content %}
+
+  <h1>Blog Posts</h1>
+
+  {% for post in post_list %}
+      <div>
+          <h2><a href="{% url 'blog:post-detail' post.id %}">{{ post.title }}</a></h2>
+          <p>{{ post.content }}</p>
+          <p><strong>Author:</strong> {{ post.author }}</p>
```

Comment:
Ensure that the 'author' attribute exists on the 'post' object. If the attribute name is different, adjust it accordingly.